Source/Fuse.Reactive.Bindings/Instance.uno

```diff
@@ -324,12 +324,36 @@ namespace Fuse.Reactive
 			}
 		}
 		
-		//contains only the added nodes (not all data items)
+		/**
+			A list of all items which have been added to the Instantiator. This only includes the items
+			within the range of Offset+Limit.
+			
+			This list should only be modified via the InsertNew, RemoveAt, and RemoveAll functions.
+		*/
 		List<WindowItem> _windowItems = new List<WindowItem>();
```

Comment:
Encapsulation would make it harder to write those other functions. This is more of a note for that if you need to modify this value you shouldn't do it unless you know what those other functions already do. I'm never clear on where/how to make such notes.